layers/+lang/python/config.el

```diff
@@ -28,3 +28,6 @@
   "Automatically set pyenv version from \".python-version\".
 
 Possible values are `on-visit', `on-project-switch' or `nil'.")
+
+(add-hook 'python-mode-hook
+          (lambda () (set (make-local-variable 'comment-inline-offset) 2)))
```

Comment:
Thanks for reminding. First time to use `use-package`, still not quite familiar with it. Will modify this PR soon.